ports/zeldadoi/port.json

```
@@ -0,0 +1,38 @@
+{
+    "version": 3,
+    "name": "zeldadoi.zip",
+    "items": [
+      "Zelda Dungeons of Infinity.sh",
+      "zeldadoi"
+    ],
+    "items_opt": [],
+    "attr": {
+      "title": "The Legend of Zelda: Dungeons of Infinity",
+      "porter": [
+        "t0b10",
+        "Shallow",
+        "JanTrueno",
+        "Jeod"
+      ],
+      "desc": "This game is a procedurally-generated rogue-like 'Link to the Past' dungeon crawler. It's not a rom hack, it was created in Gamemaker Studio from scratch using the original assets. Many new features and assets were added to this game, but special care was put in to ensure it still looks, feels, and plays like the original game.\nThe goal of the game is to descend through the many enemy-infested dungeons to defeat Link's nemesis, Aghanim. Along the way you will find many weapons, items, and treasures that will help you along your quest.",
+      "desc_md": null,
+      "inst": "Ready to run",
+      "inst_md": null,
+      "genres": [
+        "action",
+        "adventure",
+        "rpg"
+      ],
+      "image": "screenshot.png",

```

PortMaster will take care of this :)
```suggestion
      "image": null,
```

---

Hopefully, this is the last change.